File: src/main/java/com/sequenceiq/cloudbreak/controller/json/BlueprintRequest.java
package com.sequenceiq.cloudbreak.controller.json;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.sequenceiq.cloudbreak.controller.doc.ModelDescriptions.BlueprintModelDescription;
import com.wordnik.swagger.annotations.ApiModel;
import com.wordnik.swagger.annotations.ApiModelProperty;

@ApiModel
@JsonIgnoreProperties(ignoreUnknown = true)
public class BlueprintRequest extends BlueprintBase {
    @ApiModelProperty(value = BlueprintModelDescription.URL)
    private String url;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/BlueprintRequest.kt
package com.sequenceiq.cloudbreak.api.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.sequenceiq.cloudbreak.doc.ModelDescriptions.BlueprintModelDescription
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

@ApiModel
@JsonIgnoreProperties(ignoreUnknown = true)
class BlueprintRequest : BlueprintBase() {
    @ApiModelProperty(value = BlueprintModelDescription.URL)
    var url: String? = null
}
